### DISCUSSION TITLE
Is the following groupBy query possible without using `$queryRaw`?

### DISCUSSION THREAD

**Discussion by justinsmid** [prisma/prisma#13939] [Q&A]
I have a particular groupBy query that I have working using `$queryRaw`, but I cannot figure out how / if I can write this query without using queryRaw.

My (**simplified**) schema is:

```prisma
model Media {
  id Int @id @default(autoincrement())
  
  file   File @relation(fields: [fileId], references: [id])
  fileId Int  @map("file_id")
  
  @@map("media")
}

model File {
  id               Int    @id @default(autoincrement())
  fileNameOriginal String @map("file_name_original")
  fileSize         Int    @map("file_size")
  
  media Media[]
  
  @@map("file")    
}
```

Where I have media items that are linked to a file.

The query I'm working on should get all the media items that are linked to **duplicate** files, the names of said duplicate files, and the number of times each file is duplicated. My current (**working**) raw query is as follows:

```sql
await db.$queryRaw`
    SELECT
        media.id AS id,
        file.file_name_original AS fileNameOriginal,
        COUNT(file.id) AS duplicateCount
    FROM
        media AS media
    INNER JOIN
        file AS file
        ON media.file_id = file.id
    GROUP BY
        file.file_name_original,
        file.file_size
    HAVING
        COUNT(file.id) > 1
    ORDER BY
        media.id ASC
`
```

**Comment by austincrim** [comment]
Hey @justinsmid 👋,

This query is not yet possible in Prisma. We are looking at supporting more use cases with `_count` and filtering, this seems like a one we should support eventually.

It would be great if you could add your thoughts/use case to this [existing issue](https://github.com/prisma/prisma/issues/8935) around `_count`!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
